Lacrit/assignment4/Islands.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <vector>
+#include "Islands.h"
+
+int rowInd[] = {-1, 1, 0, 0};
+int colInd[] = {0, 0, 1, -1};
+
+Islands::Islands(std::vector<std::vector<bool>> grid) 
+{
+	this->grid = grid;
+	this->rows = grid.size();
+	this->cols = grid[0].size();
```

Comment:
Minor comment, grid rows can potentially be of different size, ideally you want to either check that they are all of the same size or handle the case where they are not.